pkg/workloads/spark_job/spark_util.py

```diff
@@ -223,8 +223,30 @@ def ingest(ctx, spark):
 
     for raw_column_name in ctx.raw_columns.keys():
         raw_column = ctx.raw_columns[raw_column_name]
-        expected_types = CORTEX_TYPE_TO_ACCEPTABLE_SPARK_TYPES[raw_column["type"]]
         actual_type = input_type_map[raw_column_name]
+
+        if actual_type not in SPARK_TYPE_TO_CORTEX_TYPE.keys():
```

Comment:
when would this `if` be `true`?

Reply:
if it somehow inferred a spark type we did not support, like timestamp